lib/iex/lib/iex/introspection.ex

```diff
@@ -115,13 +126,42 @@ defmodule IEx.Introspection do
         end
         :ok
       else
-        :not_found
+        if has_callback?(mod, fun, arity) do
+          :behaviour_found
+        else
+          :not_found
+        end
       end
     else
       :no_docs
     end
   end
 
+  defp has_callback?(for, fun) do
```

Comment:
I used `for` to follow the same convention as in https://github.com/uesteibar/elixir/blob/6c75771bf88a0ceda31cca25dfedd9c9636f7ac0/lib/iex/lib/iex/introspection.ex#L481, but I agree it's not the best name, I'll find a better one :)